tests/integration/test_data/pip_multiple/fetch_deps_sha256sums.json

```diff
@@ -0,0 +1,5 @@
+{
+    "pip/aiowsgi-0.7.tar.gz": "9fad305430fa39d54cd10a47a367a98c47424e5f176774d443bcf48aa5b5be33",
+    "pip/github.com/quay/appr/appr-external-gitcommit-58c88e4952e95935c0dd72d4a24b0c44f2249f5b.tar.gz": "1847a6d5d9c4ba2e71f5607c5481fa2a2979ba43c57a548c81ae3e727d16999b",
```

Comment:
nitpick: according to the code that's in this commit, this should already be `gitcommit:58c88e4952e95935c0dd72d4a24b0c44f2249f5b`